k8s/lib.sh

```diff
@@ -69,6 +69,9 @@ k8s::remove::network() {
   do
     tc filter del dev $default_interface ${d} || true
   done
+
+  rm -rf /var/run/cilium/cilium.pid
+  "${SNAP}/usr/bin/cilium-dbg" cleanup --all-state --force
```

Comment:
What happens if this fails? Will the remove-hook continue or will it abort the uninstall process?

Reply:
Good catch, thanks!